src/main/resources/messages/CoroutineStacksBundle.properties

```diff
@@ -1,2 +1,5 @@
 no.java.debug.process.is.running=No Java debug process is currently running
 should.be.stopped.on.a.breakpoint=The debug process should be stopped on a breakpoint
+coroutine.stacks.could.not.be.built=Coroutine Stacks could not be built
+select.dispatcher=Select Dispatcher:  
+number.of.coroutines.active={0} Coroutines Active
```

Comment:
Let's omit the `Active` word. We also need to add proper ending for examples like: 1 Coroutine, 2 Coroutine**s**